### PR TITLE
Search fixups

### PR DESCRIPTION
**Pull request by jherr**
Changed the styling, gave some informational behavior, updated the screencast.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -186,5 +186,11 @@ pre, code {
   left: 100px;
```
Comment (ananavati) [inline src/style.css]: left: 315px to center the search box.

File: src/style.css [hunk 2]
```diff
@@ -186,5 +186,11 @@ pre, code {
...
   width: 50%;
   position: absolute;
-  font-size: large;
+  font-size: 1rem;
+  line-height: 1.25;
+  padding: 0.5rem 0.75rem;
+  color: #55595c;
+  background-color: #fff;
```
Comment (ananavati) [inline src/style.css]: background-color #273c4c

File: README.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ Electrify is a tool for analyzing the module tree of
 for catching large and/or duplicate modules which might be either bloating up
 your bundle or slowing down the build process.
 
-![screencast](https://cloud.githubusercontent.com/assets/360041/18318796/ea0ddae4-74d7-11e6-89cb-08e02e4b1683.gif)
+![screencast](./img/screencast.gif)
```
Comment (ananavati) [inline README.md]: can we please increase the framerate of the `gif` or else it looks very slow in the screencast

File: src/style.css (hunk 1 above)
Comment (ananavati) [inline src/style.css]: screen shot 2016-09-21 at 5 28 43 pm

**Comment by jherr** [comment]
Ok. Updated. But I took a different approach with the centering.

**Comment by ananavati** [comment]
👍

**Pull request merged**